frontend/src/client/schemas.gen.ts

```diff
@@ -234,6 +245,119 @@ export const HTTPValidationErrorSchema = {
+            title: 'Pid'
+        },
+        hostname: {
+            type: 'string',
+            title: 'Hostname'
+        },
+        user: {
+            type: 'string',
+            title: 'User'
+        },
+        acquired_at: {
+            type: 'number',
+            title: 'Acquired At'
+        },
+        expires_at: {
+            type: 'number',
+            title: 'Expires At'
+        },
+        version: {
+            type: 'string',
+            pattern: '(\\d+(\\.\\d+){0,2}|\\d+\\.\\d+\\.[a-z0-9]+\\+[a-z0-9.]+)',
+            title: 'Version',
+            default: '0.3.3.dev1+g0cc9b0624'
+        }
+    },
+    type: 'object',
+    required: ['pid', 'hostname', 'user', 'acquired_at', 'expires_at'],
+    title: 'LockInfo',
+    description: 'Represents a .fmu directory lock file.'
+} as const;
+
+export const LockStatusSchema = {
+    properties: {
+        is_lock_acquired: {
+            type: 'boolean',
+            title: 'Is Lock Acquired',
+            description: 'Whether the current session holds the write lock.'
+        },
+        lock_file_exists: {
+            type: 'boolean',
+            title: 'Lock File Exists',
+            description: 'Whether a lock file exists.'
+        },
+        lock_info: {
+            anyOf: [
+                {
+                    '$ref': '#/components/schemas/LockInfo'
+                },
+                {
+                    type: 'null'
+                }
+            ],
+            description: 'Contents of the lock file, if available and readable.'
+        },
+        lock_status_error: {
+            anyOf: [
+                {
+                    type: 'string'
+                },
+                {
+                    type: 'null'
+                }
+            ],
+            title: 'Lock Status Error',
+            description: 'Error message if checking lock status failed.'
+        },
+        lock_file_read_error: {
+            anyOf: [
+                {
+                    type: 'string'
+                },
+                {
+                    type: 'null'
+                }
+            ],
+            title: 'Lock File Read Error',
+            description: 'Error message if reading the lock file failed.'
+        },
+        last_lock_acquire_error: {
+            anyOf: [
+                {
+                    type: 'string'
+                },
+                {
+                    type: 'null'
+                }
+            ],
+            title: 'Last Lock Acquire Error',
+            description: 'Error message from the last attempt to acquire the lock.'
+        },
+        last_lock_release_error: {
+            anyOf: [
+                {
+                    type: 'string'
+                },
+                {
+                    type: 'null'
+                }
+            ],
+            title: 'Last Lock Release Error',
+            description: 'Error message from the last attempt to release the lock.'
+        }
```

Comment:
I just added `last_lock_refresh_error` in [here](https://github.com/equinor/fmu-settings-api/pull/143).
```suggestion
        },
        last_lock_refresh_error: {
            anyOf: [
                {
                    type: 'string'
                },
                {
                    type: 'null'
                }
            ],
            title: 'Last Lock Refresh Error',
            description: 'Error message from the last attempt to refresh the lock.'
        }
```